flow/ProtocolVersion.h

```diff
@@ -0,0 +1,123 @@
+/*
+ * ProtocolVersion.h
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2018 Apple Inc. and the FoundationDB project authors
```

Comment:
I guess this should probably be 2019 on a new file, though I'm not particularly familiar with what our thoughts are on updating these in general.

Reply:
Changed it to 2019 for now.